apps/vs-code-react/src/app/export/export.tsx

```diff
@@ -6,23 +6,14 @@ import './export.less';
 import { Navigation } from './navigation/navigation';
 import { Text } from '@fluentui/react';
 import type { OnErrorFn } from '@formatjs/intl';
+import { getReactQueryClient } from '@microsoft/logic-apps-designer';
 import { useCallback } from 'react';
 import { IntlProvider } from 'react-intl';
-import { QueryClient, QueryClientProvider } from 'react-query';
+import { QueryClientProvider } from 'react-query';
 import { useSelector } from 'react-redux';
 import { Outlet, useOutletContext } from 'react-router-dom';
 
-const queryClient = new QueryClient({
-  defaultOptions: {
-    queries: {
-      refetchInterval: false,
-      refetchIntervalInBackground: false,
-      refetchOnWindowFocus: false,
-      refetchOnReconnect: false,
-      refetchOnMount: true,
```

Comment:
In this one, the only difference from the `getReactQueryClient` is that the `refetchOnMount` its true here and over there its false. I do remember we faced some issues when we set it up to false

Reply:
I guess it's not actually needed over here to use the designer query client, I'll go ahead and revert this

Reply:
Sure, I guess in the future if we needed I can take a further look on how to properly make it work it without the refetchOnMount